src/event_loop.rs

```diff
@@ -88,21 +88,18 @@ impl<T> EventLoopBuilder<T> {
     /// ***For cross-platform compatibility, the [`EventLoop`] must be created on the main thread,
     /// and only once per application.***
     ///
-    /// Attempting to create the event loop on a different thread, or multiple event loops in
-    /// the same application, will panic. This restriction isn't
-    /// strictly necessary on all platforms, but is imposed to eliminate any nasty surprises when
-    /// porting to platforms that require it. `EventLoopBuilderExt::any_thread` functions are exposed
-    /// in the relevant [`platform`] module if the target platform supports creating an event loop on
-    /// any thread.
+    /// Attempting to create the event loop on a different thread, or multiple
+    /// event loops in the same application, will panic. This restriction
+    /// isn't strictly necessary on all platforms, but is imposed to eliminate
+    /// any nasty surprises when porting to platforms that require it.
+    /// `EventLoopBuilderExt::any_thread` functions are exposed in the relevant
+    /// [`platform`] module if the target platform supports creating an event
+    /// loop on any thread.
```

Comment:
I'd suggest using standard sections headers, something like this:
```suggestion
    /// # Panics
    ///
    /// Attempting to create the event loop on a different thread, or multiple
    /// event loops in the same application, will panic. This restriction
    /// isn't strictly necessary on all platforms, but is imposed to eliminate
    /// any nasty surprises when porting to platforms that require it.
    /// `EventLoopBuilderExt::any_thread` functions are exposed in the relevant
    /// [`platform`] module if the target platform supports creating an event
    /// loop on any thread.
```